### ISSUE TITLE
[OC 8.2.0] Where is the document app ?

### ISSUE THREAD

**Issue by Wikinaut** [owncloud/documents#571]
After having updated the server to 8.2.0, I do not see the "document" app ? What happened ?

**Comment by fuco809** [comment]
check if folder owncloud/apps/documents exists (if not download newest app version).
after upgrade to OC8.2 you have to reenable the app in the Apps area!

**Comment by Wikinaut** [comment]
The App is not listed when I click onto APP+ icon.

**Comment by Wikinaut** [comment]
I thought, and still think, it was (is?) one of the highlights of OC.

**Comment by VicDeo** [comment]
@Wikinaut thanks for your feedback, I really appreciate it :)
I've published 8.2 compatible version just now, It should appear in your apps section soon.

**Comment by Wikinaut** [comment]
@VicDeo Pls. let me know, when it should be visible, and where exactly. I have double-checked all the entries under APP+ and can still not find the Document-App. Something is broken.

**Comment by VicDeo** [comment]
@Wikinaut I did my job (publishing).
it doesn't depend on me any more.  As soon as your app cache is refreshed you'll see the app.
If you would like to install it on your own just download https://apps.owncloud.com/content/download.php?content=168711&id=3&tan=68221732
and extract into your apps directory.

**Comment by Wikinaut** [comment]
@VicDeo my question is, in which part (Nicht aktiviert, Multimedia, Productivity, Game, Tool) the Documents App will become listed.

**Comment by VicDeo** [comment]
@Wikinaut I guess `Nicht aktiviert` as it is not enabled

**Comment by Wikinaut** [comment]
@VicDeo 

Clarification request: is the document app designed to be shown in one of the mentioned sections, or not ?

On my server (fresh installation 8.2.0 plus re-copying the backup copy of the config file which I made before the update- this is my usual way of updating, tell me if this is wrong), OC "homepage", I cannot see the document app. There are other apps visible (many of course deactivated), but I cannot see the document app.

In other words: is the document app now an app which must be looked up and installed _manually_ from the https://apps.owncloud.com/ ?

**Comment by RealRancor** [comment]
@Wikinaut If published correctly at the appstore the app should be installable via the built-in appstore as every other app.

**Comment by Wikinaut** [comment]
@RealRancor ist sie aber nicht!  App is _not_ listed in the built-in Appstore.

**Comment by VicDeo** [comment]
none of the apps is `designed to be shown in one of the mentioned sections` unless someone uploads the app to https://apps.owncloud.com/
https://apps.owncloud.com/content/show.php/Documents?content=168711
	`ownCloud 8.2` means that OC8.2 compatible version is uploaded. That's it.

**Comment by RealRancor** [comment]
@VicDeo AFAIK you also need to set a specific version for each available version in the appstore so that it shows up in the build-in appstore of oC. Have seen something similar with another app where the version wasn't set.

**Comment by fwolfst** [comment]
Hi there, that's what I stumbled upon as well. As is https://doc.owncloud.org/server/8.2/admin_manual/configuration_files/collaborative_documents_configuration.html seems off. Any hints about how to refresh the apps cache (although i made a fresh installation just hours ago, so I assume the cache would have been created just now)?

**Comment by RealRancor** [comment]
@fwolfst Refeching the app cache won't work here as https://api.owncloud.com/v1/content/data?version=8x2x0x0&filter=all&page=0&pagesize=100 is not listing the app and so it can't appear in your built-in appstore.

**Comment by VicDeo** [comment]
Displayed at the Productivity tab with 8.2.0. for me.

**Comment by RealRancor** [comment]
@VicDeo Yes, now it also appears at https://api.owncloud.com/v1/content/data?version=8x2x0x0&filter=all&page=0&pagesize=100

**Comment by Wikinaut** [comment]
I reported that. Finally solved. (I close this issue). Please next time make sure to avoid such problems.

The document app is still not working opening my existing documents in Firefox, but this is another issue.

**Comment by RealRancor** [comment]
> Please next time make sure to avoid such problems.

It should be obvious that 3rd party apps are not available instantly after an major upgrade. So to avoid such problems please make sure that the app you're using is available for your version. This can be easily checked at https://apps.owncloud.com.
